da/mock/common.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from typing import TypeAlias, List
+
+from da.verifier import Attestation, DABlob
+from da.common import Certificate, NodeId
+
+Id: TypeAlias = bytes
+
+@dataclass
+class Metadata:
+    index: int
+    app_id: Id
+
+@dataclass
+class AttestationMessage:
+    attestation: Attestation
+    metadata: Metadata
+    blob_id: Id # TODO: might be included in the Attestation itself.
+
+@dataclass
+class DABlobMessage:
+    blob: DABlob
+    metadata: Metadata
+
+@dataclass
+class CertificateMessage:
+    certificate: Certificate
+    metadata: Metadata
```

Comment:
I think this is the most important part. A storage/retrieval specification on how to store and query this data is what I feel is missing.